src/python/WMCore/Services/AlertManager/AlertManagerAPI.py

```diff
@@ -0,0 +1,123 @@
+"""
+
+AlertManagerAPI - send alerts to MONIT AlertManager via API calls
+"""
+
+from __future__ import division
+from datetime import tzinfo, timedelta, datetime
+import socket
+import time as _time
+import json
+
+from WMCore.Services.pycurl_manager import RequestHandler
+
+
+class AlertManagerAPI(object):
+
+    """
+
+    configDict: the WMCore service configuration
+    alertName: a unique name for the alert
+    severity: low, medium, high
+    labels: a customizable dict of key:value pairs used for routing the alert, ex. "tag": "microservices"
+            must be configured in AlertManager by MONIT team
+    end: how many minutes until the alarm is silenced
+    generatorURL: this URL will be sent to AlertManager and configured as a clickable "Source" link in the web interface
+    annotations: a customizable dict of key:value pairs to be appended to the alert, typically:
+                 "hostname": "the hostname where the alert generated" (appended automatically)
+                 "summary": "a short description of the alert"
+                 "description": "a longer informational message with details about the alert"
+
+    AlertManager JSON format reference: https://www.prometheus.io/docs/alerting/latest/clients/
+    [
+      {
+        "labels": {
+        "alertname": "<requiredAlertName>",
+        "<labelname>": "<labelvalue>",
+        ...
+      },
+        "annotations": {
+          "<labelname>": "<labelvalue>",
+        ...
+      },
+        "startsAt": "<rfc3339>", # optional, will be current time if not present
+        "endsAt": "<rfc3339>",
+        "generatorURL": "<generator_url>"
+      },
+    ]
+    """
+
+    def __init__(self, configDict, alertName, severity, labels, end, generatorURL, annotations):
+        self.alertManagerUrl = configDict.get("alertManagerUrl", "http://cms-monitoring.cern.ch:30093/api/v1/alerts")
```

Comment:
Erik, please do not hardcode anything. The code is in public repository and this leads to leaking our infrastructure details to entire world (the internal ports we use).

Instead, I suggest that code should throw exception if such parameter is not provided.

Reply:
My bad, I had that there for my testing w/o the config object. Removed it.